SonarQube.TeamBuild.PreProcessor/SonarLintAnalyzerProvider.cs

```diff
@@ -96,8 +96,6 @@ private static bool TryCreateRuleset(ISonarQubeServer server, TeamBuildSettings
                 }
                 else
                 {
-                    // TODO: decide on an appropriate error message once we know whether could
-                    // happen in the 1.1 release.
                     logger.LogError(Resources.SLAP_InvalidRulesetReturned);
```

Comment:
I can't think of a scenario in which we'd hit this line of code in practice. However, it's testing and working so I'll leave it in case there are scenarios we haven't thought of.